web/styles/right_sidebar.css

```diff
@@ -517,13 +510,52 @@ $user_status_emoji_width: 24px;
     }
 }
 
-.right-sidebar-shortcuts {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
+.buddy-list-user-link,
+.invite-user-shortcut {
+    margin-right: var(--width-simplebar-scroll-hover);
+    padding-left: var(--right-sidebar-left-spacing);
+    border-radius: 4px;
+
+    &:hover {
+        background: var(--color-background-sidebar-action-heading-hover);
+        box-shadow: inset 0 0 0 1px var(--color-shadow-sidebar-row-hover);
+        color: var(--color-text-sidebar-action-heading-hover);
+    }
+
+    .buddy-list-user-link-text,
+    .invite-user-link {
+        display: grid;
+        grid-template-rows: var(--line-height-sidebar-row-prominent);
+        text-decoration: none;
+        align-items: center;
+        color: var(--color-text-sidebar-action-heading);
+
+        &:active {
+            transition: transform 80ms;
+            transform: scale(0.96);
+        }
+    }
+}
+
+.buddy-list-user-link {
+    margin-left: var(--right-sidebar-toggle-width-offset);
+}
+
+.invite-user-link {
+    grid-template-columns: auto 1fr;
```

Comment:
Is there a reason not to include this with the rest of the `.invite-user-link` definition above? Also, `1fr` should be `minmax(0, 1fr)` to prevent grid blowouts.

Reply:
The one above this is combined with `.buddy-list-user-link-text` but I'm assuming you're meaning the other one below? Because they definitely can be combined so I'll do that. Also updated to `minmax(0, 1fr)`